style.css

```diff
@@ -37,11 +37,15 @@ label {
   flex-direction: row;
   justify-content: space-between;
   align-items: center;
-  max-width: 30.5rem;
+  width: 100%;
   top: 1rem;
   background-color: #ffff;
 }
 
+.nav-menu {
+  display: none;
+}
+
```

Comment:
- [ ] great job and kudo to you, as you have done a great job on this milestone I can say your project is great and look exactly like Figma template but according to this project requirement, we are to check if this mobile design still looks like expected, but as you can see the screenshot below .showing the mobile menu icon not align with the name Navigation logo compare with the original design onmobile, kindly fix the mobile menu icon or The hamburger button should be aligned with the navigation name logo, to make your website look professional.
- [ ] Your design
![Screenshot 2023-07-20 212844](https://github.com/melkamu372/Portfolio-mobile-first/assets/92458236/2a257e66-c2e6-4d75-b8e0-b56bf66abdfa)

- [ ] figma design
![Screenshot 2023-07-20 212738](https://github.com/melkamu372/Portfolio-mobile-first/assets/92458236/e588dcb3-db99-43f3-b870-1d6991b355d0)